backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/Edge.java

```diff
@@ -58,89 +53,74 @@
 
     private List<Position> routingPoints;
 
+    private Position sourcePosition;
+
+    private Position targetPosition;
+
```

Comment:
The simple fact that those positions are not used in the frontend makes me strongly believe that you are just using those fields are not useful.

Reply:
> Why are those necessary? Why not use the first and last routing points?

The first and the last routing points cannot be used here because depending on how elements are moved between each other, these routing points can change, and will get closer to one of their node corners, and thus, they will eventually overlap.

Reply:
> The simple fact that those positions are not used in the frontend makes me strongly believe that you are just using those fields are not useful.

I don't need them in the front end, but I need them to be persisted in database.